policies/nru.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "../src/cache.h"
+
+class NRU : public Cache{
+    private:
+        bool* recentlyUsed;
+        long long* allUsed;
```

Comment:
```suggestion
        long long* nrecentlyUsed;
```

Or something similar. ```allUsed``` is confusing. I was expecting ```allUsed``` to be a bool storing if a set is full or not.